dojo.js

```diff
@@ -771,182 +777,184 @@
 	}
 
 
-	if(has("dojo-combo-api")){
-		req.combo = req.combo || {add:noop};
-		var	comboPending = 0,
-			combosPending = [],
-			comboPendingTimer = null;
-	}
-
+	if (!has("foreign-loader")) {
+		if(has("dojo-combo-api")){
+			req.combo = req.combo || {add:noop};
+			var	comboPending = 0,
+				combosPending = [],
+				comboPendingTimer = null;
+		}
+		
+
+		// build the loader machinery iaw configuration, including has feature tests
+		var	injectDependencies = function(module){
```

Comment:
`var	injectDependencies` -> `var injectDependencies`  (I know you didn't introduce this, but might as well fix this weird tab formatting while we're here